device_calendar.code-workspace

```diff
@@ -1,11 +1,11 @@
 {
 	"folders": [
-		{
-			"path": "device_calendar"
-		},
 		{
 			"path": "device_calendar/example",
```

Comment:
oops i'll need to delete this/add to .gitignore

Reply:
Yeah, I was thinking that as well

Reply:
Dunno how to do it...